FRD-1415-new-release-antifraudsdk-v1.1.4

spvagelis opened the pull request:
- Updated the AntifraudSDK.podspec file for version 1.1.4-test
- Updated the CHANGELOG.md file for version 1.1.4
- Updeted the Package.swift file for version 1.1.4-test with new checksum
- Updated the README.md file for version 1.1.4
- Uploaded the new AntifraudSDK.xcframework

spvagelis committed:
- Updated the AntifraudSDK.podspec file for version 1.1.4-test
- Updated the CHANGELOG.md file for version 1.1.4
- Updeted the Package.swift file for version 1.1.4-test with new checksum
- Updated the README.md file for version 1.1.4
- Uploaded the new AntifraudSDK.xcframework

cjd1884 commented on CHANGELOG.md:
```diff
@@ -1,3 +1,8 @@
+
+### Fixed
+- Location updates are now started on the main thread to ensure correct Core Location behavior and improve stability.
+
```
@spvagelis Since I see in the changes above the use of Swift 6.2.1 compiler instead of 6.1, add also the following:

```MARKDOWN
### Added
- Support Swift 6.2.1
```

spvagelis replied:
Done.

spvagelis committed:
- Set podspec to version 1.1.4
- Update the version of Swift from 6.1 to 6.2.1
- Added support Swift 6.2.1 to CHANGELOG.md file
- Update url in Package.swift file

cjd1884 reviewed (approved):
@spvagelis Well done! 🎉
